content/community/changelog/app-frontend/v4/migrating-from-v3/_index.en.md

```diff
@@ -136,3 +136,40 @@ In addition to changing the version, you should also remove the links to the thi
       }
     ```
 4. Fix any [breaking changes](/community/changelog/app-frontend/v4) affecting your app, including upgrading the nuget packages to version 8.0.
+
+## Opt-in to Altinn AI for tracks migration
```

Comment:
Maybe add a link to this section from the breaking changes page here: https://docs.altinn.studio/community/changelog/app-frontend/v4/#tracks-are-no-longer-supported